lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -559,6 +560,40 @@ class ReportUIFeatures {
     }
   }
 
+  /**
+   * Tab + Enter for quick jump document.activeElement to the first interesting element.
+   */
+  _setUpQuickTabFocus() {
+    const firstContentSelectors = [
+      {
+        selector: '.lh-audit--fail summary',
+        label: 'Jump to first failing audit.',
+      },
+      {
+        selector: '.lh-audit--average summary',
+        label: 'Jump to first average audit.',
+      },
+      {
+        selector: '.lh-audit--audit summary',
+        label: 'Jump to first audit.',
+      },
+    ];
+
+    const {selector, label} = firstContentSelectors.find(({selector}) => {
+      const el = this._document.querySelector(selector);
+      return Boolean(el && el instanceof HTMLElement);
+    }) || {selector: '.lh-category-header .lh-gauge__wrapper', label: 'Jump to first category.'};
```

Comment:
why not just put this last in the array?

Reply:
Because this asserts that the variable will always be non-null (if find returns null, the `||` is the backup)

Reply:
seems like sort of a roundabout way of asserting that :)

Reply:
Fair - I refactored it.